### PR TITLE
updated Readme

### PR DESCRIPTION
**Pull request by nateusmc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
+# MOOD CLOUD *tentative*
+
+## ABOUT THIS PROJECT
+In this age of school bullying and shootings, sometimes it is not clear what a student's emotional state is for teachers to see. This app will give teachers and counselors access to the emotional state of their students. Students could update their emotional status every day where only the teachers have access to. If a teacher realizes a student is in a critical or depressed emotional condition, the teacher now have an opportunity to reach out immediately. In addition, student have access to record events of any incidents that made them hurtful - physically and/or emotionally. Teachers, too, will be notified and have access to these incidents as they are being logged.
```
Comment (visaudi) [inline README.md]: "students have access"
Could also be a more clear sentence.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,56 @@
...
+
+## CICD
+* Mocha/Chai
+* Jest
+
+
+##
+
+
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```
Comment (visaudi) [inline README.md]: Can we get rid of the createReactApp stuff?

Comment (nateusmc) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (nateusmc) [reply README.md]: Done

**Pull request merged**